x/leverage/keeper/keeper.go

```diff
@@ -28,19 +31,25 @@ func NewKeeper(
 	paramSpace paramtypes.Subspace,
 	bk types.BankKeeper,
 	ok types.OracleKeeper,
-) Keeper {
+) (Keeper, error) {
 	// set KeyTable if it has not already been set
 	if !paramSpace.HasKeyTable() {
 		paramSpace = paramSpace.WithKeyTable(types.ParamKeyTable())
 	}
 
-	return Keeper{
-		cdc:          cdc,
-		storeKey:     storeKey,
-		paramSpace:   paramSpace,
-		bankKeeper:   bk,
-		oracleKeeper: ok,
+	tokenRegCache, err := simplelru.NewLRU(100, nil)
```

Comment:
This argument should probably be a  descriptive constant

Reply:
yes, good idea, adding it.